themes/digital.gov/layouts/communities/list.html

```diff
@@ -68,6 +68,9 @@ <h2 class="usa-card__heading">{{- $e.Params.dg_shortname | markdownify -}}</h2>
 
           {{/* Page Title */}}
           <h1>Other Communities of Practice</h1>
+          <div class="deck"> 
+            "These communities are not managed by Digital.gov, but we want to highlight them because they overlap with <a class="usa-link" href="{{about}}">our mission.</a>
```

Comment:
Hey @bonnieAcameron, Thanks for providing some description for the other communities.
I suggest we update the markup to pass the build errors:

`These communities are not managed by Digital.gov, but we want to highlight them because they overlap with <a class="usa-link" href="{{- "/about/" | relURL -}}">our mission.</a>`

This will link to the about page and renders without any errors.

Reply:
That build went through, but alignment got skewed. Will add `div` to see if that helps

## Top
<img width="1189" alt="cop pg test top" src="https://user-images.githubusercontent.com/19961574/202248496-3a5a315e-c136-46f7-a0b0-f1a41b9ca112.png">


## Bottom 
<img width="1164" alt="cop pg test bottom" src="https://user-images.githubusercontent.com/19961574/202248519-bf319ede-76e7-481e-b721-a7ccb8a56365.png">